Cargo.toml

```diff
@@ -20,11 +20,13 @@ clap = { version = "4.2.1", features = ["derive", "env"] }
 expanduser = "1.2.2"
 http = "*"
 hyper = { version = "0.14", features = ["full"] }
+lazy_static = "1.4"
 maligned = "0.2.1"
 mime = "0.3"
 ndarray = "0.15"
 ndarray-stats = "0.5"
 num-traits = "0.2.15"
+prometheus = { version = "0.13", features = ["process"] }
```

Comment:
There's an [axum](https://github.com/tokio-rs/axum/tree/main/examples/prometheus-metrics) example that uses metrics-exporter-prometheus. I wonder how the two compare.

Reply:
There's also https://github.com/prometheus/client_rust which seems to have historically been related to the prometheus crate I'm currently using but has since been adopted as the official client library. Might make sense to switch to this.

Reply:
That's not a problem. Axum can [pass state](https://docs.rs/axum/latest/axum/extract/struct.State.html) to handlers or [middleware](https://docs.rs/axum/latest/axum/middleware/index.html#accessing-state-in-middleware).

Perhaps the main consideration is whether the non-official library will continue to be maintained.